README.md

```diff
@@ -13,9 +13,11 @@ In the Google online script editor, select the `Resources` menu item and choose
 #### Creating a service account
 The easiest way to use this library is to create a Google Service Account for your application and give it read/write access to your datastore. Giving a service account access to your datastore is like giving access to a user's account, but this account is strictly used by your script, not by a person.
 
+In the [Firebase admin console](https://console.firebase.google.com) create a new Firestore project with the default security settings. Open the project and in the settings menu (cog) of the  project overview menu go to the "User and permissions", from here a service account can be generated if one doesn't already exist. 
```

Comment:
I like that you added this. It's good to just explain it all here. I had linked (in the next paragraph) to Google's instructions for creating a service account, and those instructions link to the service account page directly. I liked linking to Google's instructions because then if they're pages ever change, it's on them to update their documentation. But I think it's safe to say that they're process isn't going to change much (or at all), so we're in the clear to just add the instructions here.

But I think if we do that, we should change all of these paragraphs up. The next paragraph will need to change too because it covers similar material to this paragraph and we shouldn't give the same instructions twice (that could be confusing).